code/modules/supply/supply_pack.dm

```diff
@@ -44,18 +46,18 @@
 
 /datum/supply_packs/New()
 	. = ..()
-	if(contains)
-		manifest += "<ul>"
-		for(var/path in contains)
-			if(!path)
-				continue
-			var/atom/movable/AM = path
-			manifest += "<li>[initial(AM.name)]</li>"
-			//Add the name to the UI manifest
-			ui_manifest += "[initial(AM.name)]"
-		manifest += "</ul>"
-	else
-		ui_manifest = list(manifest)
```

Comment:
This area is a revert from the emergency shuttle PR, testing the effects / changes currently but there should not be a difference